app/coffee/PersistenceManager.coffee

```diff
@@ -44,12 +44,20 @@ module.exports = PersistenceManager =
 					return callback(new Error("web API response had no valid doc version"))
 				if !body.pathname?
 					return callback(new Error("web API response had no valid doc pathname"))
+				# fix up any broken docs that are already in mongo
```

Comment:
It will be an internal reference in an open source repo, but maybe worth linking to https://github.com/overleaf/issues/issues/2162 here?

Reply:
Reference to original issue now added.